code/game/objects/items/cards_ids.dm

```diff
@@ -174,10 +174,9 @@
 	if(!isliving(user))
 		return
 	if(!user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+		to_chat(user, "<span class='warning'>You can't do that right now!</span>")

```

Comment:
canUseTopic already handles this message.

Reply:
fukj